website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -104,3 +104,13 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt --help`: Lists the commands available for dbt<br />
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
+
+### Run dbt commands
+
+When you're managing your dbt project, it's important to understand that dbt commands are categorized into the following types:
+
+- **Data platform write commands** &mdash; Commands such as `dbt build` and `dbt run` that perform write operations to your data platform. These commands are limited to one invocation at any given time. This is to prevent any potential conflicts, such as overwriting the same table in your data platform, at the same time. For example, you can't run `dbt build` and `dbt run` at the same time.
+
+- **Data platform read commands** &mdash; Commands such as `dbt parse` and `dbt source snapshot-freshness` that don't write to your platform. These commands aren't limited to one invocation at any given time and you can run multiple invocations in parallel. For example, you can run `dbt parse` and `dbt source snapshot-freshness` at the same time.
```

Comment:
@dichenqiandbt do we want to say 'read' or is 'non-write' the correct way to phrase this? I've opted for 'read'.

Reply:
A little bit tricky here, as some commands won't interactive with data warehouse, e.g. deps, docs. Technically write, read only and data warehouse irrelevant, three types of commands.

But I'm neutral on this, if read is better for customer to understand, that's totally fine.